go.mod

```diff
@@ -1,23 +1,62 @@
 module github.com/kubernetes-csi/csi-lib-utils
 
-go 1.16
+go 1.19
 
 require (
 	github.com/container-storage-interface/spec v1.5.0
```

Comment:
Can you update this to 1.7.0?

Reply:
Why 1.17? k8s 1.26.0 is at 1.19

Reply:
github.com/container-storage-interface/spec v1.5.0 -> v1.7.0 which is the latest CSI spec release.  This is not K8s release.

Reply:
this comment is about github.com/container-storage-interface/spec

Reply:
Then I need to fix the previous 1.25.0 first

Reply:
But the go version for github.com/container-storage-interface is 1.18? https://github.com/container-storage-interface/spec/blob/release-1.7/go.mod

Reply:
This should be fine.  It's not possible for all dependency libs to use the exact same go version.

Reply:
I see this is still `github.com/container-storage-interface/spec v1.5.0`.  Is there any issue updating to 1.7.0?

Reply:
Oh sorry I somehow was thinking you want me to update the go version to 1.17.0, changing it now. And i am still trying to fix the component base.